config/config.txt

```diff
@@ -106,7 +106,7 @@ ANIMAL_DELAY 0
 
 REVIVE_GRACE_PERIOD 3000
 
-
+FUN_ALLOWED

```

Comment:
put an 1 or 0 (ergo true or false) after the defined config.

Reply:
Ok then.

Reply:
The default here should be false.

Reply:
FUN_ALLOWED in this case means the marines are allowed to have fun, and as such, dab without consequence.

Reply:
Precisely. We don't want arm-removing dabbing in normal rounds.

Reply:
the check for delimb is `if(!FUN_ALLOWED)` since the default is FUN_ALLOWED 1 people can dab to their hearts content.

Reply:
Yes, and I'm saying `FUN_ALLOWED` should be off by default.

Reply:
done it that way. dabbing is not available unless FUN_ALLOWED = 1

Reply:
I'm thinking that the changelog should be removed also, as this is no longer something available to everyone unless specifically allowed by an admin.